yal/mainmenu.py

```diff
@@ -58,6 +58,15 @@ async def get_suggested_choices(self, parent_topic_links={}):
 
         return suggested_choices
 
+    async def get_privacy_reminder_sent(self, whatsapp_id):
+        privacy_reminder_sent = self.user.metadata.get("privacy_reminder_sent")
+        error = await rapidpro.update_profile(
+            whatsapp_id, {"privacy_reminder_sent": "True"}, self.user.metadata
+        )
```

Comment:
I think this means that we'll be updating the profile every time we call this method, ie. even if it's already `True`, we'll still make the call to update it. Maybe we can put this request in a `if not privacy_reminder_sent:`